dynamodb/src/it/scala/zio/dynamodb/LiveSpec.scala

```diff
@@ -0,0 +1,589 @@
+import zio.dynamodb.SortKeyExpression.SortKey
+import io.github.vigoo.zioaws.{ dynamodb, http4s }
+import zio._
+import zio.dynamodb.DynamoDBQuery._
+import zio.dynamodb.ProjectionExpression._
+import zio.test.Assertion._
+import zio.test.environment._
+import software.amazon.awssdk.regions.Region
+import zio.schema.{ DeriveSchema, Schema }
+import zio.stream.ZSink
+import zio.test._
+import zio.test.TestAspect._
+
+import java.net.URI
+import scala.collection.immutable.{ Map => ScalaMap }
+
+object LiveSpec extends DefaultRunnableSpec {
+
+  val awsConfig = ZLayer.succeed(
+    config.CommonAwsConfig(
+      region = None,
+      credentialsProvider = SystemPropertyCredentialsProvider.create(),
+      endpointOverride = None,
+      commonClientConfig = None
+    )
+  )
+
+  val liveAws = http4s.default >>> config.default >>> dynamodb.live >>> DynamoDBExecutor.live
+
+  val layer: ZLayer[Any, Throwable, Has[DynamoDBProxyServer] with Has[DynamoDBExecutor]] =
+    ((http4s.default ++ awsConfig) >>> config.configured() >>> (dynamodb.customized { builder =>
+      builder.endpointOverride(URI.create("http://localhost:8000")).region(Region.US_EAST_1)
+    } >>> DynamoDBExecutor.live)) ++ (Blocking.live >>> LocalDdbServer.inMemoryLayer)
+
+  private val id       = "id"
+  private val first    = "first"
+  private val second   = "second"
+  private val third    = "third"
+  private val name     = "firstName"
+  private val number   = "num"
+  val secondPrimaryKey = PrimaryKey(id -> second, number -> 2)
+
+  private val notAdam = "notAdam"
+  private val avi     = "avi"
+  private val avi2    = "avi2"
+  private val avi3    = "avi3"
+  private val adam    = "adam"
+  private val adam2   = "adam2"
+  private val adam3   = "adam3"
+  private val john    = "john"
+  private val john2   = "john2"
+  private val john3   = "john3"
+
+  private val stringSortKeyItem = Item(id -> adam, name -> adam)
+
+  private final case class Person(id: String, firstName: String, num: Int)
+  private implicit lazy val person: Schema[Person] = DeriveSchema.gen[Person]
+
+  private val aviPerson  = Person(first, avi, 1)
+  private val avi2Person = Person(first, avi2, 4)
+  private val avi3Person = Person(first, avi3, 7)
+
+  private val aviItem  = Item(id -> first, name -> avi, number -> 1, "mapp" -> ScalaMap("abc" -> 1, "123" -> 2))
+  private val avi2Item = Item(id -> first, name -> avi2, number -> 4)
+  private val avi3Item = Item(id -> first, name -> avi3, number -> 7)
+
+  private val adamItem  = Item(id -> second, name -> adam, number -> 2)
+  private val adam2Item = Item(id -> second, name -> adam2, number -> 5)
+  private val adam3Item = Item(id -> second, name -> adam3, number -> 8, "listt" -> List(1, 2, 3))
+
+  private val johnItem  = Item(id -> third, name -> john, number -> 3)
+  private val john2Item = Item(id -> third, name -> john2, number -> 6)
+  private val john3Item = Item(id -> third, name -> john3, number -> 9)
+
+  def insertData(tableName: String) =
```

Comment:
```suggestion
  private def insertData(tableName: String) =
```
can these defs be made private?